overlay/mkcrate.nix

```diff
@@ -305,13 +305,23 @@ let
         if (( NIX_DEBUG >= 1 )); then
           set -x
         fi
-        env \
-          "CC_${stdenv.buildPlatform.config}"="${ccForBuild}" \
-          "CXX_${stdenv.buildPlatform.config}"="${cxxForBuild}" \
-          "CC_${rustHostTriple}"="${ccForHost}" \
-          "CXX_${rustHostTriple}"="${cxxForHost}" \
-          "''${depKeys[@]}" \
-          ${buildCmd}
+        dep_keys_sanitized="$(echo ''${depKeys[@]})"
```

Comment:
I don't think this should be necessary.  Can you provide a test crate?  I'm going to hit CI with a commit that works for recording dep values with spaces, but I'm not sure of the playback in the build phase.

Reply:
The crate that failed were `gdk-sys`, `gdk` and `gdkx11-sys`